src/pages/SplashPage.vue

```diff
@@ -32,74 +32,38 @@ import { useRoute, useRouter } from 'vue-router';
 import { onMounted } from 'vue';
 import { initPlugins } from 'src/composables/PluginManager';
 import PluginEvent from 'src/composables/events/PluginEvent';
-import {
-  getUserSessionToken,
-  removeUserSessionToken,
-  login,
-  initUserInformation,
-  initUserRoles,
-} from 'src/composables/UserAuthentication';
-import { useUserStore } from 'src/stores/UserStore';
+import { initUserInformation, initUserPermissions } from 'src/composables/UserAuthentication';
 
 const { t } = useI18n();
 const route = useRoute();
 const router = useRouter();
-const userStore = useUserStore();
 
 /**
- * Initialize the user for the appplication, if needed.
- * If the backend is activated in the configuration, it will check
- * that the session token is present or not. If not, it will try to login the user.
- * If already connected, it will just retrieve the user info.
+ * Initializes the user by fetching user information and permissions.
```

Comment:
In leto-modelizer, I think we don't use the third person, so `Initialize the user...`

Reply:
done